blockchain/reactor.go

```diff
@@ -114,9 +114,8 @@ func (bcR *BlockchainReactor) GetChannels() []*p2p.ChannelDescriptor {
 
 // AddPeer implements Reactor by sending our state to peer.
 func (bcR *BlockchainReactor) AddPeer(peer *p2p.Peer) {
-	if !peer.Send(BlockchainChannel, struct{ BlockchainMessage }{&bcStatusResponseMessage{bcR.store.Height()}}) {
-		// doing nothing, will try later in `poolRoutine`
-	}
+	// doing nothing, will try later in `poolRoutine`
```

Comment:
rather keep this in an `if` clause.  thats the context in which this comment is relevant.